### PR TITLE
Tip to delete existing mobilespec if it exists

### PR DESCRIPTION
**Pull request by brodybits**
before running `createmobilespec.js`

(this seems to help me avoid the quirks I documented before)

Note that this proposal is expected to conflict with PR #154, which I did not get a chance to review yet. I am proposing this tip anyway due to its importance IMHO, to capture the info and avoid losing track of it. I hope to review the documentation updates proposed in PR #154 within the next few days.

### CODE REVIEW

**Comment by janpio** [comment]
Hm, I changed the merge target to my PR branch so this is less complicated - but seems it added another commit here, thanks GitHub Conflict resolution UI. Should probably go away when merging back into the branch.

File: createmobilespec/README.md [hunk 1]
```diff
@@ -78,6 +78,9 @@ If anything is going wrong and the existing output does not help, add the `--deb
 
 ### Create the App
 
+Caution: If the generated `mobilespec` project already exists, it is recommended
+to delete the project before running `createmobilespec.js`.
```
Comment (janpio) [inline createmobilespec/README.md]:
```suggestion
to delete the project before proceeding.
```

Comment (brodybits) [reply createmobilespec/README.md]: Done (with a force push)

Comment (janpio) [reply createmobilespec/README.md]: You can actually just include those in the GitHub UI, no need to do it manually - a lot faster. (Merging via Squash+Merge will make one commit out of it anyway)

**Pull request merged**